lazy-load.js

```diff
@@ -30,11 +30,12 @@ function fetchImage(url) {
  */
 function preloadImage(image) {
   const src = image.dataset.src;
+  const srcset = image.srcset;
   if (!src) {
```

Comment:
You need to check here for srcset too. Not sure if there are other instances that need the same.